ui-particles-angular/projects/ui-particles-angular/src/scss/gio-mat-palettes.scss

```diff
@@ -163,12 +163,18 @@ $mat-method-palette: mat.define-palette(
     put: #fbf2f2,
     get: #ebf7ff,
     delete: #fff5f4,
+    option: #f4f6ff,
```

Comment:
to be added here too : https://github.com/gravitee-io/gravitee-ui-particles/blob/c14f80ce33f5f6bb87a1dd2ec6dbf8bcf061ff15/ui-particles-angular/projects/ui-particles-angular/src/scss/theme/color/gio-palettes.stories.scss#L53

Reply:
to have color inside storybook